modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -4,22 +4,16 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, esb, studio, raise, error
 
-This Core component generates a xref:mule-error-concept.adoc[Mule Error], as if a failure had occurred, which allows you to customize its description and type.
+This Core component generates a Mule Error, as if a failure had occurred, which allows you to customize its description and type.
 
-Note that you can only use core types, such as `SECURITY` or custom ones. You cannot raise an
-error from another module like `FILE:FILE_NOT_FOUND`.
+Note that you only this component to raise core runtime errors, such as `MULE:SECURITY`, `MULE:CONNECTIVITY`, etc, or custom ones.
+You cannot raise an error from another connector or module like `FILE:FILE_NOT_FOUND`, `JSON:INVALID_SCHEMA`.
 
-Core errors must use the implicit namespace, while custom ones are forced to declare one.
-The custom error namespace must not be used elsewhere in your app, meaning that you cannot
-use `HTTP` as a namespace if you are using HTTP operations.
-Once you declare a custom namespace by using it in a `raise-error`, you can use it in other
+For core runtime errors type, you must use the implicit namespace and identifier, you can only customize its description message. +
+For custom error types, you are forced to declare one namespace. The namespace of an error type can help you identify the origin of an error. +
```

Comment:
`one namespace` -> `a namespace`?

Reply:
`can help you identify the origin of an error` -> `should help you identify the origin of an error`